src/analysis/normed_space/basic.lean

```diff
@@ -918,6 +918,17 @@ real.norm_of_nonneg (norm_nonneg _)
 @[simp] lemma nnnorm_norm [normed_group α] (a : α) : nnnorm ∥a∥ = nnnorm a :=
 by simp only [nnnorm, norm_norm]
 
+-- `metric.tendsto_at_top` says `∃ N, ∀ n ≥ N, ...` while here we use `∃ N, ∀ n > N, ...`,
```

Comment:
I'd rather have the same inequalities as in metric spaces, i.e., `∀ n ≥ N` in both cases. What about having a `tendsto_at_top` and a `tendsto_at_top'` with the strict inequality in both files? Also, in the other file it is formulated with `semilattice_sup`. I guess your version with strict inequalities holds assuming `semilattice_sup` and `no_top_order`.

Reply:
I've fixed both of these issues.